cookiecutter/wrapper-template/{{cookiecutter.wrapper_id}}/pyproject.toml

```
@@ -0,0 +1,38 @@
+[tool.poetry]
+name = "{{cookiecutter.wrapper_id}}"
+version = "0.0.1"
+description = "`{{cookiecutter.wrapper_id}}` is a Meltano utility extension."
+authors = ["{{ cookiecutter.admin_name }}"]
+license = "Apache 2.0"
+
+[tool.poetry.dependencies]
+python = "^3.8"
+structlog = "^21.2.0"
+PyYAML = "^6.0.0"
+pydantic = "^1.9.0"
+click = "^8.1.3"
+typer = "^0.6.1"
+
+[tool.poetry.dev-dependencies]
+black = "^22.3.0"
+isort = "^5.10.1"
+flake8 = "^3.9.0"
+
+[build-system]
+requires = ["poetry-core>=1.0.8"]
+build-backend = "poetry.core.masonry.api"
+
+packages = [
+    { include = "{{cookiecutter.library_name}}" }
+    # uncomment this if you want to include static assets
+    # { include = "files_{{cookiecutter.wrapper_id}}.*" }
+]
+
+# If you need to static assets with your extension, you can add them here.
+# Under a directory path of "files_{{cookiecutter.wrapper_id}}/"
+#include = [
+#    "files_{{cookiecutter.wrapper_id}}/somefile.thing"
+#]
+
+[tool.poetry.scripts]
+{{cookiecutter.library_name}} = '{{cookiecutter.library_name}}.main:app'
```

As per my prior comment, an additional 'swap-in' CLI entrypoint could be provided here:

```suggestion
{{cookiecutter.library_name}} = '{{cookiecutter.library_name}}.main:app'
{{cookiecutter.library_name}}-run = '{{cookiecutter.library_name}}.main:invoke'
```